src/components/Navbar/SettingsDrawer/SettingsBodySkeleton.module.scss

```diff
@@ -0,0 +1,42 @@
+.skeletonContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.title {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  margin: var(--spacing-xxsmall);
+}
+
+.titleSkeleton {
+  width: calc(3 * var(--spacing-mega));
+  height: var(--spacing-medium);
+}
+
+.rowSkeleton {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.inputRow {
+  margin-bottom: var(--spacing-large);
```

Comment:
Do you have `style-lint` installed? because `margin-bottom/up` is a physical value and we are meant to use [logical values](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties). `yarn lint:scss` should have run on committing which would have auto-fixed this but for some reason it seems it was not run so you might want to manually run `yarn lint:scss` and push again.

Reply:
dono, but will run it manually and push.
Just curious, I understand the value of logical margins for right and left, but what difference it will make for up/bottom?